README.md

```diff
@@ -361,5 +344,29 @@ You will also find the following intermediate files:
 
 ## Debugging
 
-To debug some docker logs, set `DEBUG_DOCKER=true`.
-To debug the localstack docker container startup, set `DEBUG_LOCALSTACK=true`.
+To enable debug level logging we are using the [debug][debug] library to create the log lines.  
+There is also an environment variable `ENABLE_LAMBDA_LOGGING` available to
```

Comment:
I think this env var can be removed now? There are some `afterEach` hooks below that reference and I think they can be cleaned up too.

Reply:
Oops, I forgot to go update these.